Workbooks/SecurityStatus.json

```diff
@@ -99,8 +99,8 @@
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources",
             "value": [
-              "/subscriptions/44e4eff8-1fcb-4a22-a7d6-992ac7286382",
-              "/subscriptions/82931e73-05c6-4da8-a666-bc4a7dd1bd3e"
+              "/subscriptions/82931e73-05c6-4da8-a666-bc4a7dd1bd3e",
```

Comment:
Is this a specific subscription you own? If so, I don't think it will work for customers who are querying other subscriptions?